extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -610,7 +616,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
   };
 
   private searchContactsOnGoogle = async (query: string, knownContacts: ContactPreview[]): Promise<EmailProviderContact[]> => {
-    if (this.googleContactsSearchEnabled) {
+    if (await this.googleContactsSearchEnabled) {
```

Comment:
I suggest to remove this condition, as we already check this before calling `searchContactsOnGoogle`

Reply:
Please remove